.github/workflows/scheduled.yml

```diff
@@ -396,3 +396,72 @@ jobs:
           name: join-fuzzer-failure-artifacts
           path: |
             /tmp/join_fuzzer_repro
+
+
+  presto-java-aggregation-fuzzer-run:
+    runs-on: 8-core
+    container: ghcr.io/facebookincubator/velox-dev:presto-java
+    timeout-minutes: 120
+    if: "$${{ github.event_name != 'pull_request' }}"
```

Comment:
```suggestion
    if: ${{ github.event_name != 'pull_request' }}
```

Reply:
Of course this will prevent it from running in this PR so comment it out for now?